tests/bluetooth/audio/ascs/mocks/mock-sections.ld

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+SECTIONS
```

Comment:
why is this necessary?

Reply:
The unit test linker script seem to be very simple. It does not create sections i.e. for Bluetooth (`common-rom-bt.ld`) so that `bt_gatt_service_static` and `bt_conn_cb` iterable sections are not there.
ASCS uses those 2 sections to register connection callbacks (`bt_conn_cb`) and register service instance (`bt_gatt_service_static`).

Thus I added this script that will create the missing sections